packages/rug/src/values.js

```diff
@@ -182,6 +184,18 @@ const contrast = (colorName) => (
   getContrastColor({dark: color.charcoal, light: color.white})(brandColors[colorName])
 )
 
+export const gradient = reduce({
+  up: '0deg',
+  right: '90deg',
+  down: '180deg',
+  left: '270deg'
```

Comment:
One of the main gradients in the designs is actually not represented in this. Which is top but slightly left,  ~350deg.